esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -728,7 +728,7 @@ bool WsWuInfo::hasSubGraphTimings()
 {
     try
     {
-        WuScopeFilter filter("stype[subgraph],props[stat]");
+        WuScopeFilter filter("depth[3],stype[subgraph],stat[TimeElapsed],nested[0]");
```

Comment:
depth 3 prevents nested subgraphs being matches, filtering to a single statistic enables the iterator to know the values are stored globally, nested[0] prevents generating matches for any nested scopes.